metrics/web/lib/dashboard/presentation/widgets/build_number_scorecard.dart

```diff
@@ -2,19 +2,21 @@ import 'package:flutter/material.dart';
 import 'package:metrics/common/presentation/metrics_theme/model/metrics_theme_data.dart';
 import 'package:metrics/common/presentation/metrics_theme/widgets/metrics_theme.dart';
 import 'package:metrics/dashboard/presentation/strings/dashboard_strings.dart';
+import 'package:metrics/dashboard/presentation/view_models/build_number_scorecard_view_model.dart';
 import 'package:metrics/dashboard/presentation/widgets/no_data_placeholder.dart';
-import 'package:metrics/dashboard/presentation/widgets/text_metric.dart';
+import 'package:metrics/base/presentation/widgets/scorecard.dart';
 
-/// Widget that displays the [TextMetric] with the build number metric.
+/// Widget that displays the [Scorecard] with the build number metric.
 ///
 /// Applies the text styles from the [MetricsThemeData.metricWidgetTheme].
 /// If the [buildNumberMetric] is either `null` or equal to 0, displays the [NoDataPlaceholder]
-/// and applies the text styles from [MetricsThemeData.inactiveWidgetTheme] to it.
-class BuildNumberTextMetric extends StatelessWidget {
-  final int buildNumberMetric;
+/// and applies the text styles from the [MetricsThemeData.inactiveWidgetTheme] to it.
```

Comment:
```suggestion
```
I would probably remove that as we don't apply it here - so it just default NoDataPlaceholder behavior.